cfme/fixtures/pytest_selenium.py

```diff
@@ -740,6 +740,28 @@ def go_to_fixture(fixtureconf, browser):
     force_navigate(page_name)
 
 
+class ContextWrapper(object):
```

Comment:
This class seems evil.

If all __getattr__ is going to do is raise an exception, it really should raise AttributeError. But it's looking at something's Keys, and so it should probably raise KeyError, but the interface is an item lookup, so nobody should be doing attribute accesses in the first place.

I see the desire to list the available keys in the exception message, but it seems like subclassing dict, or wrapping a dict and implementing `__iter__` and `__contains__` would be more pythonic, especially if the correct exceptions are raised on `__getitem__` misses and the `__getattr__` override goes away.

Setting self.__dict__ to a passed in also seems like unnecessary roughness, and again points to subclassing dict (`self.update(data)`) or wrapping a dict (`self._data = data`) as more appropriate solutions.